frontend/src/metabase/visualizations/visualizations/Table.tsx

```diff
@@ -435,12 +444,13 @@ class Table extends Component<TableProps, TableState> {
           )}
         >
           <img
+            data-testid="Table-all-fields-hidden-image"
```

Comment:
What is the up-to-date data-testid naming convention?

Reply:
Good question! Usually I use `lowercase-dash-case` but can I see in the codebase that sometimes we prefix the testid with the capitalized visualization component like you did here (`Table-xyz`), so this is fine